File: examples/thesis/figs/py/latexVSword.m
function latexVSword()
t = 0:0.1:10;
latex = t.^0.5;
word  = 1./(7-t)-(1/7);
plot(t,latex,t,word,'linewidth',2);
ylim([0,5]);
yticks([])
xticks([])
set(gca,'fontsize',18);
xlabel('Document Complexity \& Size','interpreter','latex')
ylabel('Effort \& Time','interpreter','latex')
legend('location','nw',{'\LaTeX','MS Word'},'interpreter','latex')
print(gcf,'latex-vs-word.eps','-depsc');

File: examples/full/thesis/figs/py/latex-vs-word.py
import numpy as np
import matplotlib.pyplot as plt
plt.rc('text',usetex=True)
plt.rc('font',family='serif',serif=['computer modern roman'],size=18)
t = np.linspace(0,2,256)
latex = t**(1/3)
word = t**3
plt.plot(t,latex)
plt.plot(t,word)
eps = 0.02
plt.ylim(0-eps,2+eps)
plt.xlim(0-eps,2+eps)
plt.yticks([])
plt.xticks([])
# plt.axis('equal')
plt.xlabel('Document Complexity \& Size')
plt.ylabel('Effort \& Time')
plt.legend(['\LaTeX','MS Word'])
plt.savefig('../latex-vs-word.eps')
# plt.show()
